client/me/help/style.scss

```diff
@@ -1,29 +1,48 @@
-.help__header {
-	margin-top: 15px;
-	@include breakpoint( "<660px" ) {
-		font: 300 16px/21px $sans;
-	}
+.help__support-link {
+	display: flex;
 }
 
-.help__support-link {
-	margin-bottom: 10px;
-	@include breakpoint( ">660px" ) {
-		float: left;
-		width: 33%;
+.help__support-link div {
+	padding-right: 32px;
+}
+
+.help__support-link h2 {
```

Comment:
Can we target the classes for all these generic html elements? Like `.help__support-link-title`.